app/views/posts/_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= form_with url: user_posts_path, scope: :post do |form| %>
+  <div class="form-group">
+    <%= form.text_field :title, class:"form-control", placeholder:"Post Title" %>
+  </div>
+  <br />
+  <div class="form-group" style: "display: block">
+    <%= form.text_area :text, class:"form-control", placeholder:"Text", rows: 4, required: true %>
+  </div>
+  <br />
+<<<<<<< HEAD
+  <div class="d-flex justify-content-center">
+     <%= form.submit "Post", class:"btn btn-secondary" %>
+=======
+ <div class="d-flex justify-content-center">
+    <%= form.submit "Post", class:"btn btn-secondary" %>
+>>>>>>> c38d85248c21e753f3d214779742cc222469800d
```

Comment:
Please remove ```<<<<< HEAD ====== >>>>>>``` this after resolving the conflicts to make the code look clean 👍🏻